src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -250,11 +250,37 @@ public static class TextCatalog
         /// <param name="catalog">The text-related transform's catalog.</param>
         /// <param name="columns">Pairs of columns to remove stop words on.</param>
         /// <param name="language">Langauge of the input text columns <paramref name="columns"/>.</param>
```

Comment:
>/// <param name="language">Langauge of the input text columns <paramref name="columns"/>.</param> [](start = 8, length = 97)

did you want to reference the sample from here?

Reply:
Am I doing this right?

---
In reply to: [250825853](https://github.com/dotnet/machinelearning/pull/2236#discussion_r250825853) [](ancestors = 250825853)